spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -153,13 +161,17 @@ def main(argv: Sequence[str]):
     if arguments.centerline_soft == 1 and method != 'fitseg':
         printv("ERROR: -centerline-soft can only be used with '-method fitseg'.", type='error')
 
+    # Extrapolation option can only be used with fitseg method
+    if arguments.extrapolation == 1 and method != 'fitseg':
+        printv("ERROR: -extrapolation can only be used with '-method fitseg'.", type='error')
+
     # Gap between slices
     interslice_gap = arguments.gap
 
     param_centerline = ParamCenterline(
         algo_fitting=arguments.centerline_algo,
         smooth=arguments.centerline_smooth,
-        minmax=True,
+        minmax=arguments.extrapolation,
```

Comment:
Before, the default was `True`. Now, we use the default of `arguments.extrapolation`, which is `0`, which == `False`.

Is this change intentional? Or do we essentially need to reverse the value?

- `extrapolation=0` --> `minmax=True`
- `extrapolation=1` --> `minmax=False`

An easy way to do this could be to add a `not` here:

```suggestion
        minmax=(not arguments.extrapolation),  # NB: 'extrapolation=0' --> 'minmax=True' and vice versa
```

```python
>>> extrapolation = 0
>>> minmax = (not extrapolation)
>>> print(minmax)
True
```